File: Data Structures & Algorithms/duplicate-integer/submission-4.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> a = new HashSet<>();
        for (int i : nums){
            if (a.contains(i)){
                return true;
            }
            else{
                a.add(i);
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-4.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        sd = {}
        td = {}
        for i in s:
            if i not in sd:
                sd[i] = 1
            else:
                sd[i] += 1
        for i in t:
            if i not in td:
                td[i] = 1
            else:
                td[i] += 1
        return sd == td
            

